symphony/content/content.publish.php

```diff
@@ -1687,7 +1687,7 @@ private function prepareAssociationsDrawer($section)
 
                         // Create link
                         $link = SYMPHONY_URL . '/publish/' . $as['handle'] . '/';
-                        $a = new XMLElement('a', $as['name'] . ' <span>(' . count($entry_id) . ')</span>', array(
+                        $a = new XMLElement('a', $as['name'] . ' <span>(' . count($entry_ids) . ')</span>', array(
```

Comment:
The two lines you changed ;) I am working on a rebased PR which include this commit.